test/integration/suspend-resume.spec.js

```diff
@@ -69,15 +69,15 @@ describe('QIX Suspend/Resume', () => {
       .then((a) => { app = a; })
       // set a dummy property that we don't save:
       .then(() => app.setAppProperties({ test: true }))
-      .then(() => global.session.suspend())
+      .then(() => session.suspend())
       .then(() => expect(suspended.calledOnce).to.equal(true))
       .then(() => global.session.resume())
       .then(() => app.getAppProperties())
       // verify that we have reconnected to a fresh app, since we never saved
       // this property it shouldn't exist in a new one:
       .then(props => expect(props.test).to.equal(undefined))
       .then(() => global.deleteApp(app.id))
-      .then(() => global.session.close())
+      .finally(() => session.close())
```

Comment:
`finally()` is not part of the ES6 spec (yet, AFAIK there are discussions but nothing decided), we may want to note that we're using the bluebird superset here.

Reply:
Good point, I'll remove it here (and create a new issue about removing all bluebird references).